hugo.toml

```diff
@@ -337,6 +337,7 @@ defaultContentLanguage = 'en'
     keepQuotes = true
 
 [params]
+  releaseVersion = '1.10.0'
```

Comment:
`latest_release` would be suggested the param name,  as it's used in `The latest release` chapter of the web page.

Reply:
I think `latestRelease` is better? Align the hump naming

Reply:
SGTM.